packages/auth/src/connection/Connection.ts

```diff
@@ -694,10 +694,15 @@ export class Connection extends EventEmitter<ConnectionEvents> {
     this.#machine = createActor(machine)
 
     // emit and log all transitions
-    this.#machine.subscribe(state => {
-      const summary = stateSummary(state.value as string)
-      this.emit('change', summary)
-      this.#log(`⏩ ${summary} `)
+    this.#machine.subscribe({
+      next: state => {
```

Comment:
There are two signatures for the `subscribe` method, one accepts `Observer<T>` object and the other accepts listener functions as separate arguments. I've opted for switching to the object signature here because I find it easier to reason about but we could switch to passing multiple functions if we choose.

Docs: https://stately.ai/docs/actors#error-handling  
Source: https://github.com/statelyai/xstate/blob/main/packages/core/src/createActor.ts#L322-L350